source/fuzz/fuzzer_util.cpp

```diff
@@ -1616,6 +1616,38 @@ bool IdUseCanBeReplaced(opt::IRContext* ir_context,
     return false;
   }
 
+  switch (use_instruction->opcode()) {
```

Comment:
It's Section 2.16.2 of the SPIR-V spec, "Validation Rules for Shader Capabilities", that says:

"All <id> used for Scope <id> and Memory Semantics <id> must be of an OpConstant."

So this switch statement could be wrapped inside an if statement that checks whether the Shader capability is enabled. This can be obtained by looking up the feature manager from `ir_context`.

Reply:
Done.